drivers/dai/intel/ssp/ssp.c

```diff
@@ -9,6 +9,7 @@
 #include <stdint.h>
```

Comment:
Capitalize Intel and Zephyr in commit message.

typo: refference -> reference

Reply:
fixed